docs/events/api/timeline-api-update.asciidoc

```diff
@@ -0,0 +1,108 @@
+[[timeline-api-update]]
+=== Add a note to an existing Timeline
+
+Add a note to an existing Timeline or Timeline event.
+
+==== Request URL
+
+`PATCH <kibana host>:<port>/api/note`
+
+==== Request body
+
+A JSON object defining the note content to be added.
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+|`note` | <<eventNotes-obj, eventNote>> a|A note added to a specific Timeline event
+
+* `timelineId`: the `savedObjectId` of an existing Timeline the note is linked to
+* `eventId`: required if the note is linked to a specific Timeline event.
```

Comment:
Suggesting minor changes:
- Remove the period at the end of "event"
- Capitalize the first letter after every colon (i.e., "the", "required", and "note")